src/schema/bookmarks.ts

```diff
@@ -161,7 +162,7 @@ export const typeDefs = /* GraphQL */ `
       The query to search for
       """
       query: String!
-    ): SearchBookmarksSuggestionsResults!
+    ): SearchBookmarksSuggestionsResults! @auth
```

Comment:
bookmark suggestions should be behind auth if bookmarks search is, userId was required either way